R/countRNAseqData.R

```diff
@@ -459,8 +459,8 @@ getSplitCountCacheFile <- function(sampleID, settings){
 #' @export
 countSplitReads <- function(sampleID, fds, NcpuPerSample=1, genome=NULL, 
                     recount=FALSE, keepNonStandardChromosomes=TRUE,
-                    bamfile=bamFile(fds[,sampleID]),
-                    pairedend=pairedEnd(fds[,sampleID]),
+                    bamfile=bamFile(fds[,sampleID,,drop=FALSE]),
```

Comment:
So this is kind of user facing code here. So not sure why we need it. I dont want to polute the code with things we do not need. 

One Idea that could happen, is that `fds` is not a fraserdataset object but rather a summarizedexperiment object. Then the default with subsetting is `drop=TRUE` which could the underlying problem